package.json

```diff
@@ -18,7 +18,7 @@
     "test:coverage-unit": "TS_NODE_FAST=true NODE_ENV=mocha LOG_LEVEL=OFF nyc mocha --opts mocha.opts $(find src/test \\( -name '*.ts' \\) ! -path '*a11y*' ! -path '*/routes/*')",
     "test:codacy-upload": "cat ./coverage-report/lcov.info | codacy-coverage",
     "test:codecov-upload": "codecov -f ./coverage-report/lcov.info",
-    "test:smoke": "./bin/run-smoke-tests.sh",
+    "test:smoke": "echo disabled due to testing breaking change to journey",
```

Comment:
Why did you disable it?

Reply:
That's strange I didn't make any changes to that file

Reply:
You merged demo into your branch. Watch more carefully what Git operations you doing. Check PR before passing it for review.